docs/concepts/tokens/fungible-tokens/rippling.md

```diff
@@ -9,92 +7,54 @@ labels:
+# The Default Ripple Flag
 
-### Specifics
+The Default Ripple flag is an account level setting that enables rippling for all incoming trust lines. The default setting of the Default Ripple flag is false. Issuers must enable this flag for their customers to be able to send tokens to one another.
 
-The No Ripple flag makes certain paths invalid, so that they cannot be used to make payments. A path is considered invalid if and only if it enters **and** exits an address node through trust lines where No Ripple has been enabled for that address.
+The Default Ripple setting doesn’t affect trust lines created by your account, only trust lines others open to your account. If you change the setting, trust lines that were created before the change keep their existing No Ripple settings. You can use a TrustSet transaction to change the No Ripple setting of a trust line to match your account’s new default.
 
-[{% inline-svg file="/docs/img/noripple-06.svg" /%}](/docs/img/noripple-06.svg "Diagram demonstrating that No Ripple has to be set on both trust lines by the same address to do anything")
-
-
-## The Default Ripple Flag
-
-The **Default Ripple** flag is an account setting that enables rippling on all _incoming_ trust lines by default. Issuers MUST enable this flag for their customers to be able to send tokens to each other.
-
-The Default Ripple setting of your account does not affect trust lines that you create; only trust lines that others open to you. If you change the Default Ripple setting of your account, trust lines that were created before the change keep their existing No Ripple settings. You can use a [TrustSet transaction][] to change the No Ripple setting of a trust line to match your address's new default.
+# The No Ripple Flag
 
+Accounts that do not want to allow rippling can set the No Ripple flag. If you set the No Ripple flag on two of its trust lines, other accounts cannot use those trust lines to ripple payments through your account.
 
 ## Using No Ripple
-<!--{# TODO: move these things into their own tutorials #}-->
-
-### Enabling / Disabling No Ripple
-
-The No Ripple flag can only be enabled on a trust line if the address has a positive or zero balance on that trust line. This is so that the feature cannot be abused to default on the obligation the trust line balance represents. (Of course, you can still default by abandoning the address.)
 
-To enable the No Ripple flag, send a [TrustSet transaction][] with the `tfSetNoRipple` flag. You can disable the No Ripple flag (that is, allow rippling) with the `tfClearNoRipple` flag instead.
+The No Ripple flag can only be enabled on a trust line if the address has a positive or zero balance. This is so that the feature cannot be abused to remove the obligation the trust line balance represents.
 
+To enable the No Ripple flag, send a `TrustSet` transaction with the `tfSetNoRipple` flag. To disable the No Ripple flag, send a `TrustSet` transaction with the `tfClearNoRipple` flag.
```

Comment:
This is a good place to link to the reference documentation—people who want to know more can click through to find more details on how that works.

```suggestion
To enable the No Ripple flag, send a [TrustSet transaction][] with the `tfSetNoRipple` flag. To disable the No Ripple flag, send a TrustSet transaction with the `tfClearNoRipple` flag.
```